static/styles/themes/night_mode.scss

```diff
@@ -376,21 +376,11 @@ on a dark background, and don't change the dark labels dark either. */
         }
     }
 
-    li.active-filter,
-    li.active-sub-filter {
-        background-color: hsla(199, 33%, 46%, 0.2);
-    }
-
-    .top_left_row:hover,
-    .bottom_left_row:hover,
-    #stream_filters li.highlighted_stream,
-    #user_presences li:hover,
-    #user_presences li.highlighted_user {
-        background-color: hsla(136, 25%, 73%, 0.2);
-    }
+    --active-filter: hsla(199, 33%, 46%, 0.2);
+    --sidebar-item-hover: hsla(136, 25%, 73%, 0.2);
 
     #stream_filters li.active-sub-filter:hover {
-        background-color: hsla(136, 25%, 73%, 0.5);
+        background-color: var(--sidebar-item-hover);
     }
```

Comment:
Ideally `night_mode.scss` should just need to change the variables; it should not need to re-specify where they are used. But the corresponding rule in `left-sidebar.scss` actually specifies a different color right now. Which suggests that maybe the name needs to be more specific?

Reply:
I'd argue that this rule is actually undesirable, so I opened #15666.